### ISSUE TITLE
value of count cannot be computed for ssh security groups

### ISSUE THREAD

**Issue by deitch** [hashicorp/terraform-aws-vault#152]
When using the `vault-cluster`, if I set the allowed inbound ssh security groups, I get:

```
module.vault.aws_security_group_rule.allow_ssh_inbound_from_security_group_ids: aws_security_group_rule.allow_ssh_inbound_from_security_group_ids: value of 'count' cannot be computed
```

This is the same issue as fixed via #68 for the allowed security groups, just related to the allowed _ssh_ security groups.

The sample terraform is:

```terraform
module "vault" {
  source = "hashicorp/vault/aws//modules/vault-cluster"
  version = "0.11.3"
  # lots of other stuff
    allowed_ssh_security_group_ids = ["${var.ssh_groups}"]
}
```

`consul-cluster` already has a fix for this, see [here](https://github.com/hashicorp/terraform-aws-consul/commit/455e43ce1b53caa78e74adeede1f3d4e67555c05)

**Comment by brikis98** [comment]
Thanks for reporting. Most of these sorts of bugs go away with Terraform 0.12. So instead of putting in a workaround using an extra, hard-coded variable, I think we should wait until we upgrade this repo to 0.12, which will hopefully happen by the end of the month.

**Comment by mcalhoun** [comment]
@deitch could you see if your issue is solved by upgrading to the latest [version](https://github.com/hashicorp/terraform-aws-vault/releases/tag/v0.13.0)?

**Comment by deitch** [comment]
Confirmed. I just ran it with terraform v0.12.3 and vault module v0.13.0. The error is gone. Am I correct in stating that this wasn't a vault module fix, as much as terraform 0.12 being able to compute count correctly in a broader set of circumstances? It still cannot do it based on resources that do not exist when building its tree (or however it does it), but it can when it depends on things like vars?
